src/twisted/newsfragments/10111.feature

```diff
@@ -0,0 +1 @@
+support py3.10a5 (by fixing twisted.trial.unittest.TestCase.flushWarnings)
```

Comment:
I think that this is not hurting to much... the convention is to write the release note starting with an object path.

twisted.trial.unittest.TestCase.flushWarnings is part of the release message , but warnAboutFunction is not included in the release notes... and I think that this PR also fixed warnAboutFunction ... and this is public API